src/Sulu/Component/Content/Types/Rlp/Strategy/TreeStrategy.php

```diff
@@ -42,6 +42,13 @@ public function __construct(
         );
     }
 
+    public function getChildPart($resourceSegment)
+    {
+        $resourceSegmentParts = explode('/', $resourceSegment);
```

Comment:
ok but then use the "functionality" of i because it feels better than exploding and imploding:

```
        $strrpos = strrpos($path, '/');

        if (false === $strrpos) {
            self::error(sprintf(
                'Path "%s" must be an absolute path', $path
            ), true);
        }

        return substr($path, $strrpos + 1);
```